charts/kube-master/templates/cloud-controller-manager.yaml

```diff
@@ -1,5 +1,5 @@
-{{- if .Values.openstack }}
-{{- if (semverCompare ">= 1.13" .Values.version.kubernetes) }}
```

Comment:
Why remove those, while the first one does nothing, the second one removes a blank line at the top. I consider it good practise to have this in front of all "conditional" blocks to avoid spurious blank lines.

There are more instances of this below. I added suggestions for a few but stopped after a while to avoid annoying repetitions. I would suggest to add `{{-` infront of every `if` and corresponding `end` block.

Reply:
I changed this everywhere.